Tasks/Common/kubernetes-common-v2/helmutility.ts

```diff
@@ -69,7 +68,12 @@ export async function getStableHelmVersion(): Promise<string> {
     try {
         const downloadPath = await toolLib.downloadTool('https://api.github.com/repos/helm/helm/releases/latest');
         const response = JSON.parse(fs.readFileSync(downloadPath, 'utf8').toString().trim());
-        return response.body.tag_name;
+        if (!response.tag_name)
```

Comment:
`response.body.tag_name`?

Reply:
it's not in body anymore

Reply:
Why do we have helmutility.ts at 2 places? Cannot we use only one?

Reply:
That's because of `kubernetes-common` and `kubernetes-common-v2`. 
v2 was created to roll out the upgrade of `vsts-task-lib` to `azure-pipelines-task-lib`

Reply:
The plan is to deprecate `kubernetes-common` once all the dependent tasks start consuming `azure-pipelines-task-lib`